Source/misdat.h

```diff
@@ -9,6 +9,12 @@
 
 namespace devilution {
 
+enum mienemy_type : uint8_t {

```

Comment:
Why name it like this? Can't we use a more meaningful name like `MissileTarget`?

Reply:
It's the current name, I don't feel like renaming things atm as you know my spelling is piss poor so it both take longer and is significantly more likely to contain mistakes. Hopefully some one else will rename things in to sensible names using our chosen code style (camel case). I like your suggested name, feel free to open a PR with it.